src/lib/pipeline.ml

```diff
@@ -173,30 +173,36 @@ module Full (Bfx: Extended_edsl.Semantics) = struct
       stringtie = bam |> Bfx.stringtie |> Bfx.save "stringtie";
       rna_bam_flagstat = bam |> Bfx.flagstat |> Bfx.save "rna-bam-flagstat";}
 
+  let rna_kallisto ~reference_build ?rna_fastqs () =
```

Comment:
Since this function doesn't do anything without RNA, I'd make it a required argument, and do the dispatch elsewhere. 

You might consider calling `run_kallisto` and stick it in the `rna_pipeline` (so the main pipeline function stays considerably smaller?